test/langtools/jdk/javadoc/doclet/testHrefInDocComment/pkg/J1.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2016, 2021, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package pkg;
+
+/**
+ *This class has <a href="{@docRoot}/pkg/J1.html#functions">various functions</
+ * see <a href="ftp://www.example.com/">FTP Site</a>,
+ *  <a href="file:///path/to/somefile">file service</a> for further information
+ *<a id="functions">various functions</a>
+ *<ul>
+ *<li>function1</li>
+ *<li>function2</li>
+ *<li>function3</li>
+ *</ul>
+ *<a id="methods">special methods</a>
+ *<ul>
+ *<li>method1</li>
+ *<li>method2</li>
+ *<li>method3</li>
+ *</ul>
+ */
+public class J1 {
+    /**
+     *fields.
+     */
+    protected Object field1;
+
+    /**
+     *<a href="{@docRoot}/pkg/J1.html#functions">Creates an instance which has
```

Comment:
Another unclosed `a` element

Reply:
Sorry, the last few characters were trimmed. I fixed it.